pkgs/development/tools/ocaml/ocaml-recovery-parser/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchFromGitHub
+, buildDunePackage
+, fix
+, menhirLib
+, menhirSdk
+}:
+
+buildDunePackage rec {
+  pname = "ocaml-recovery-parser";
+  version = "0.2.2";
+
+  useDune2 = true;
+
+  src = fetchFromGitHub {
+    owner = "serokell";
+    repo = pname;
+    rev = version;
+    sha256 = "qQHvAPNQBbsvlQRh19sz9BtfhhMOp3uPthVozc1fpw8=";
+  };
+
+  buildInputs = [
+    fix
+    menhirLib
+    menhirSdk
```

Comment:
Should these libraries be propagated?

Also, this package won’t build with OCaml < 4.08 (please set `minimalOCamlVersion`).

Reply:
Done.